tests/integration/common/s2n_test_common.py

```diff
@@ -148,6 +148,8 @@ def get_s2n_cmd(scenario):
 
     if scenario.version is Version.TLS13:
         s2n_cmd.append("--tls13")
+        if get_libcrypto() == "openssl-1.1.1":
+            s2n_cmd.extend(["-u", "default_tls13"])
```

Comment:
We should definitely be using "--curves" here for readability. "-u" is hard to guess the meaning of :)